src/components/admin/localization-statuses/LocalizationStatuses.tsx

```
@@ -0,0 +1,47 @@
+    languages: LocalizationLanguage[];
+    localizedEntity: EntityWithLocalizations<TLocalization>;
+}
+
+export const LocalizationStatuses = <TLocalization extends EntityLocalization>({
+    languages,
+    localizedEntity,
+}: LocalizationStatusProps<TLocalization>) => {
+    return (
+        <div className="statuses" data-testId="localization-statuses">
+            {languages.map((language) => (
+                <span className={`badge ${getLocalizationClassNameFromStatus(language, localizedEntity)}`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)